### ISSUE TITLE
Role Template Create/Edit UI should allow you to set "apiGroup" if desired

### ISSUE THREAD

**Issue by Oats87** [rancher/rancher#16254]
**What kind of request is this (question/bug/enhancement/feature request):**
Enhancement

**Steps to reproduce (least amount of steps as possible):**
Create Role Template in Rancher 2.1.1 UI

**Result:**
You are able to set `Verbs` and `Resources`, but `apiGroup` is implicitly set to `*`

**Desired Result:**
You should be able to set `Verbs`, `Resources`, and `apiGroups`

**Comment by janeczku** [comment]
Specific use case that requires this:
Admin needs to create a custom project role for users managing Istio resources like gateways, virtualservices etc. that live under the `networking.istio.io` ApiGroup.

**Comment by davidnuzik** [comment]
Version: Master (v2.3) (6/11/19)

I confirmed what was stated in the issue description (launch older version of Rancher, notice we cannot configure API Groups).

I then launched Master (v2.3) (6/11/19)
We now have the option to set API groups:
![image](https://user-images.githubusercontent.com/45179589/59308258-7126fc80-8c55-11e9-9ea8-9cdbd9c1aac4.png)

I was able to successfully create a roletemplate with a custom string for API Groups (root>rules>0>apiGroups>0 in my case). Also tested with multiple rules. This is of type array[string] according to the API schema (v3/schemas/policyRule) and appears to work correctly based on my testing done from a UI perspective. Create (POST) and Edit (PUT) and Delete (DELETE) all seem to work as expected.
